style.css

```diff
@@ -481,6 +464,336 @@ textarea::placeholder {
+    flex-direction: column;
+    justify-content: flex-end;
+    align-items: flex-start;
+    padding: 20px;
+  }
+
+  .rec_works_section > h2 {
+    width: auto;
+    margin: 0;
+    padding-bottom: 5px;
+  }
+
+  .rec_proj_btn.proj_2,
+  .rec_proj_btn.proj_3,
+  .rec_proj_btn.proj_4,
+  .rec_proj_btn.proj_5,
+  .rec_proj_btn.proj_6 {
+    display: none;
+  }
+
+  .rec_works_section.proj_6 {
+    background-image: url(./img/grid_img_6.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .rec_works_section.proj_5 {
+    background-image: url(./img/grid_img_5.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .rec_works_section.proj_4 {
+    background-image: url(./img/grid_img_4.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .rec_works_section.proj_3 {
+    background-image: url(./img/grid_img_3.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .rec_works_section.proj_2 {
+    background-image: url(./img/grid_img_2.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .rec_works_section.proj_1 {
+    background-image: url(./img/grid_img_1.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .rec_works_section > p {
+    width: auto;
+    font-size: 17px;
+    line-height: 24px;
+    padding-bottom: 5px;
+  }
+
+  .rec_skills_list {
+    width: auto;
+    display: flex;
+  }
+
+  .rec_proj_btn.proj_1 {
+    width: 100%;
+    margin: 0;
+  }
+
+  .rec_skills_list.proj_1 {
+    display: none;
+  }
+
+  /* About us desktop version */
+  .about_us.main {
+    width: 100vw;
+    background: #1c1a19;
+    padding: 15vh 5vw 15vh 5vw;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: flex-start;
+  }
+
+  .backgrnd_img_about_us {
+    background-image: url("./img/about_me_desk.png");
+    background-repeat: no-repeat;
+    background-position: right top;
+    background-size: contain;
+  }
+
+  .about_us > h2 {
+    font-family: 'Crete Round', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-size: 72px;
+    line-height: 82px;
+    color: #f55800;
+  }
+
+  .about_us > p {
+    font-family: 'Inter', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  .about_us_btn {
+    width: auto;
+    align-self: flex-start;
+    margin-left: 5px;
+  }
+
+  .vector-line.about_us {
+    width: 100%;
+    border: 1px solid #fff;
+    margin-top: 20px;
+  }
+}
+
+/* Button Interctions */
+button:enabled {
+  background: #f55800;
+  color: #fff;
+}
+
+button:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+button:active {
+  background: #e05304;
+  color: #f4f1f0;
+}
+
+button:disabled {
+  color: #5b5957;
+  background: #f4f1f0;
+}
+
 footer {
   width: 100vw;
   display: flex;
```

Comment:
- Hey, let's also reduce the spaces in between each icon to improve our design as compared with the Figma design:

*Your design*
![image](https://user-images.githubusercontent.com/59397159/236646321-df9d27ae-ea2f-4779-b7e4-e83032c96125.png)

*figma design*
![image](https://user-images.githubusercontent.com/59397159/236646451-54cf1a3f-e7cd-4e00-b4b6-0734288b40c4.png)